library/L1_Peripheral/pin.hpp

```diff
@@ -93,26 +98,29 @@ class Pin
   /// the job of the L1 peripheral that uses this pin to manage its own function
   /// code usage.
   ///
-  /// @param function - pin function code
+  /// @param function pin function code
   virtual void SetPinFunction(uint8_t function) const = 0;
```

Comment:
I feel like this could also return an error, do you agree?

Reply:
I agree, we could check for whether the function code is valid or not.